std_msgs/Header header
float64 n_reader               # n_reader reading
float64 n_reader_variance      # n_reader_variance reading


